src/gui/newwizard/setupwizardwindow.cpp

```diff
@@ -32,12 +53,29 @@ SetupWizardWindow::SetupWizardWindow(QWidget *parent)
         emit paginationEntryClicked(_ui->pagination->activePageIndex(), clickedPageIndex);
     });
 
+    resize(ocApp()->gui()->settingsDialog()->sizeHintForChild());
+
+    loadStylesheet();
+
     _ui->transitionProgressIndicator->setFixedSize(32, 32);
+    _ui->transitionProgressIndicator->setColor(Theme::instance()->wizardHeaderTitleColor());
 
     // handle user pressing enter/return key
     installEventFilter(this);
 }
 
+void SetupWizardWindow::loadStylesheet()
+{
+    QString path = QStringLiteral(":/client/resources/wizard/style.qss");
+
+    QFile file(path);
+    Q_ASSERT(file.exists());
+    Q_ASSERT(file.open(QIODevice::ReadOnly));
+
+    QString stylesheet = replaceCssColors(QString::fromUtf8(file.readAll()));
```

Comment:
Hm, have you checked how this looks if the stylesheet can not read for whatever reason in production (where ASSERT is silent)? In that case you will set a completely empty stylesheet. 

Maybe check if not empty before setting to keep the original one?

Reply:
The stylesheet is compiled in. How likely is it for such a file not to be loaded?

We can and should add logging, though.

Reply:
![screenshot_2022-04-27_13-56-52](https://user-images.githubusercontent.com/80399010/165513081-3e6a472b-bc2c-4d84-b699-5ae8309b7342.png)

This is what it looks like when the stylesheet cannot be loaded. It's usable.